### PR TITLE
build pin board done

### PR DESCRIPTION
**Pull request by cligh**

### CODE REVIEW

File: public/dist/js/gilt-trest.browserified.full.js [hunk 1]
```diff
@@ -29231,11 +29233,38 @@ module.exports = angular.module('request', [])
...
+      });
+
+
+
 
     }
 
     function pinSale (saleKey) {
-
+      //THEM
```
Comment (cligh) [inline public/dist/js/gilt-trest.browserified.full.js:29267]: Remove extraneous comments ("US", "THEM")

File: public/js/gilt-trest/views/pinned.js [hunk 2]
```diff
@@ -13,12 +13,25 @@ var pinnedController = function pinnedController ($scope) {
...
+
+.config(['$routeProvider', function($routeProvider) {
+  $routeProvider.when('/sales/pinned', {
+    templateUrl: 'assets/templates/views/pinned.html',
+    controller: 'pinnedController'
+  });
+}]);
 
-})();
+})();
```
Comment (cligh) [inline public/js/gilt-trest/views/pinned.js:37]: Add new line at end of file.

File: public/dist/js/gilt-trest.browserified.full.js [hunk 3]
```diff
@@ -19,6 +19,8 @@ angular.module('gilt-trest',
     $routeProvider.otherwise({redirectTo: '/sales/women'});
```
Comment (cligh) [inline public/dist/js/gilt-trest.browserified.full.js:19]: Should this file be part of the commit?

File: public/js/gilt-trest/views/pinned.js (hunk 2 above)
Comment (aluo854) [inline public/js/gilt-trest/views/pinned.js:37]: there's already a new line

File: public/dist/js/gilt-trest.browserified.full.js (hunk 1 above)
Comment (aluo854) [inline public/dist/js/gilt-trest.browserified.full.js:29267]: done

File: public/dist/js/gilt-trest.browserified.full.js (hunk 3 above)
Comment (aluo854) [inline public/dist/js/gilt-trest.browserified.full.js:19]: no

**Pull request closed without merging**